Makefile

```diff
@@ -1,8 +1,8 @@
 DOCKER:=$(shell docker-compose -v dot 2> /dev/null)
 
 ifdef DOCKER
-    APP=docker-compose exec -T app
-    TOOLS=docker-compose run --rm tools
+    APP=sudo docker-compose exec -T app
```

Comment:
sudo is not required on every platforms for this command to work: on linux, user may be added in the docker group. And macOS not needs privilege elevation at all.

It's a "hard to resolve" point...